projects/Allwinner/bootloader/release

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+SRCDIR="$BUILD/image/system/usr/share/bootloader"
+
```

Comment:
Add ` DSTDIR="$RELEASE_DIR/3rdparty/bootloader"` and you can lose the 3x repeated destination paths. :smile:

Reply:
Done, for all bootloader release scripts